internal/convenience/repository/input_raw_references_repository.go

```diff
@@ -132,3 +143,49 @@ func (r *RawInputRefRepository) FindFirstInputByStatusNone(ctx context.Context,
 	slog.Debug("First input with status NONE fetched", "row", row)
 	return &row, nil
 }
+
+func (r *RawInputRefRepository) FindByRawIdAndAppContract(ctx context.Context, rawId uint64, appContract *common.Address) (*RawInputRef, error) {
+
+	res, err := r.Db.QueryxContext(ctx,
```

Comment:
Could you please use RawInputRef to fetch the data? This avoids the need for ordered fields in parseRawInputs, allowing that method to be removed.